### ISSUE TITLE
react.semantic-ui.com with rails

### ISSUE THREAD

**Issue by coder9s** [shakacode/react_on_rails#1022]
How to install  react.semantic-ui.com with react_on_rails 10.0.2 .... i cannot use it in rails  please..help me thankyou very mumch

**Comment by justin808** [comment]
@robwise Are you using this?

**Comment by robwise** [comment]
@justin808 Yes I am, the process should be exactly the same as any other node package though. The only tricky part is if you want to use custom themes and such, but that's not because of React on Rails that's just because of the package itself.
